prov/efa/src/efa_mr.c

```diff
@@ -263,8 +263,7 @@ static int efa_mr_hmem_setup(struct efa_mr *efa_mr,
 					 "Unable to register handle for GPU memory. err: %d buf: %p len: %zu\n",
 					 err, attr->mr_iov->iov_base, attr->mr_iov->iov_len);
 				/* When gdrcopy pin buf failed, fallback to cudaMemcpy when user enables cuda xfer */
-				if (efa_mr->domain->cuda_xfer_setting != CUDA_XFER_ENABLED)
```

Comment:
do you need to modify the comment above?

Reply:
fixed in new revision